src/raw/accessor.rs

```diff
@@ -431,6 +432,9 @@ pub struct AccessorInfo {
     scheme: Scheme,
     root: String,
     name: String,
+    /// limit of batch operation
+    /// only meaningful when accessor supports batch operation
+    batch_limit: usize,
```

Comment:
How about naming to `max_batch_operations`? And maybe we should use `Option<usize>` here?